mdp_plan_exec/package.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0"?>
+<package>
+  <name>mdp_plan_exec</name>
+  <version>0.0.0</version>
```

Comment:
This will need to be the same as the other packages in this repo, currently 0.0.16.